core/kernel/user_access.c

```diff
@@ -29,8 +49,9 @@ TEE_Result copy_from_user(void *kaddr, const void *uaddr, size_t len)
 
 	uaddr = memtag_strip_tag_const(uaddr);
 	res = check_access(flags, uaddr, len);
-	if (!res)
+	if (!res) {
 		memcpy(kaddr, uaddr, len);
+	}
```

Comment:
commit "core: add more user access functions" should not add these braces. Should be done from commit "core: Apply finer-grained PAN".